src/main/java/edu/ucsb/cs156/happiercows/entities/UserCommons.java

```diff
@@ -18,16 +17,17 @@
 public class UserCommons {
   @Id
   @GeneratedValue(strategy = GenerationType.IDENTITY)
-  private long id;  
+  private long id;
+
+  @Column(name = "commons_id")
+  private long commonsId;
 
-  @Column(name="commons_id")
-  private long commonsId;  
+  @Column(name = "user_id")
+  private long userId;
 
-  @Column(name="user_id")
-  private long userId;  
+  private String username;
```

Comment:
This is not how this should be done.   Instead, you should look up the username from the user's table.  This is storing redundant data, which violates the principles of good database design.